sphinxcontrib/needs/layout.py

```diff
@@ -506,15 +507,15 @@ def meta(self, name, prefix=None, show_empty=False):
             for index, element in enumerate(data):
                 if index > 0:
                     spacer = nodes.inline(classes=["needs_spacer"])
-                    spacer += nodes.Text(", ", ", ")
+                    spacer += nodes.Text(", ")
```

Comment:
Oh ok, no second parameter anymore.
This also has not worked for me in the past.
But I had last checked with Sphinx version <4 I think....
So nice simplification

Reply:
yeah i wasn't sure when it was deprecated. I was concerned it would break some of the sphinx versions in the test matrix, but it doesn't seem to?